packages/vtex-io-logger/tests/core/instrumentation/routes/get-binding-for-route.spec.ts

```diff
@@ -16,7 +16,13 @@ describe("getBindingsForRoute", () => {
       account: "testAccount",
       workspace: "testWorkspace",
       tracer: { traceId: "testTraceId" },
+      type: "route",
     },
+    process: {
+      env: {
+        appId: process.env.VTEX_APP_ID ?? '',
+      }
+    }
```

Comment:
I think this test is not so effective, because process is inside context, what it's not true for a production runtime and you are accessing direct the env that will never exists in a test runtime. I think you can change to mock the process.env before each run and test if the function is extracting the value properly.

Reply:
updated

Reply:
updated